app/src/main/java/org/akvo/flow/api/FlowApi.java

```diff
@@ -161,5 +150,8 @@ private String getTimestamp() {
         String TIMESTAMP    = "ts=";
         String LAST_UPDATED = "lastUpdateTime=";
         String HMAC         = "h=";
+        String VERSION      = "ver=";
+        String DEVICE_ID    = "devId=";
+        String ANDROID_ID   = "androidId=";
```

Comment:
The corresponding changes for these new parameters are going into the servlet right?